clang/include/clang/Sema/SemaHLSL.h

```diff
@@ -21,6 +21,7 @@
 #include "clang/Basic/IdentifierTable.h"
 #include "clang/Basic/SourceLocation.h"
 #include "clang/Sema/Scope.h"
+#include "clang/Sema/Sema.h"
```

Comment:
What led us to include Sema.h?

Reply:
Sema.h is needed because it defines `RequireCompleteType`
`RequireCompleteType` is required because if it isn't called, then in SemaHLSL.cpp on line 625 where `isAggregate` is called, an assert will cause a crash in the compiler. Calling the function where it's called prevents this crash, but further investigation is needed because it seems like the function should have been called sometime earlier.

Reply:
If only SemaHLSL.cpp needs the header, could we move the include to SemaHLSL.cpp?

Reply:
Yes, done